src/unity/python/turicreate/toolkits/_mxnet_utils.py

```diff
@@ -80,8 +80,9 @@ def get_num_gpus_in_use(max_devices=None):
     return len(gpu_ctx)
 
 def assert_valid_num_gpus():
+    from turicreate.util import _CUDA_GPU_IDS
     num_gpus = _tc_config.get_num_gpus()
-    if _sys.platform == 'darwin' and num_gpus > 0:
+    if not _CUDA_GPU_IDS and _sys.platform == 'darwin' and num_gpus > 0:
         raise _ToolkitError('Using GPUs is currently not supported on Mac')
```

Comment:
You should probably refine the error message as well

Reply:
I thought about it. However, since you only see this message if you don't have a CUDA card, I kept it as is so as not to complicate it for the average user.